packages/mdc-card/README.md

```diff
@@ -172,6 +172,7 @@ CSS Class | Description
 --- | ---
 `mdc-card` | Mandatory, for the card element
 `mdc-card--stroked` | Removes the shadow and displays a hairline stroke instead
+`mdc-card__primary-action` | The main tappable area of the card. Typically contains most (or all) card content _except_ `mdc-card__actions`.
```

Comment:
Might want to further qualify that this is only applicable to cards that have a primary action that the main surface should trigger?

Reply:
Done